### ISSUE TITLE
macOS requires macports python 

### ISSUE THREAD

**Issue by goretkin** [alphacep/vosk-api#93]
I installed from PyPI into the miniconda distribution of python, and upon `import vosk`, I get an error

```
ImportError: dlopen(/Users/goretkin/miniconda3/lib/python3.8/site-packages/vosk/_vosk.so, 2): Library not loaded: /opt/local/Library/Frameworks/Python.framework/Versions/3.8/Python
  Referenced from: /Users/goretkin/miniconda3/lib/python3.8/site-packages/vosk/_vosk.so
  Reason: image not found
```

I'm guessing that `vosk.so` was built against a MacPorts distribution of python, since I think MacPorts puts stuff in `/opt/local`.

I don't know how to fix this, though this page has been helpful for me in the past: https://medium.com/@donblas/fun-with-rpath-otool-and-install-name-tool-e3e41ae86172

For reference,  `otool -l /Users/goretkin/miniconda3/lib/python3.8/site-packages/vosk/_vosk.so` outputs

```
...
Load command 10
          cmd LC_LOAD_DYLIB
      cmdsize 96
         name /System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate (offset 24)
   time stamp 2 Wed Dec 31 19:00:02 1969
      current version 4.0.0
compatibility version 1.0.0
Load command 11
          cmd LC_LOAD_DYLIB
      cmdsize 96
         name /opt/local/Library/Frameworks/Python.framework/Versions/3.8/Python (offset 24)
   time stamp 2 Wed Dec 31 19:00:02 1969
      current version 3.8.0
compatibility version 3.8.0
Load command 12
          cmd LC_LOAD_DYLIB
      cmdsize 48
         name /usr/lib/libc++.1.dylib (offset 24)
   time stamp 2 Wed Dec 31 19:00:02 1969
      current version 307.5.0
compatibility version 1.0.0
Load command 13
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name /usr/lib/libSystem.B.dylib (offset 24)
   time stamp 2 Wed Dec 31 19:00:02 1969
      current version 1238.60.2
compatibility version 1.0.0
Load command 14
      cmd LC_FUNCTION_STARTS
  cmdsize 16
  dataoff 5054032
 datasize 21632
Load command 15
      cmd LC_DATA_IN_CODE
  cmdsize 16
  dataoff 5075664
 datasize 832
```

**Comment by nshmyrev** [comment]
This has been fixed already, please get an updated wheel here.

https://github.com/alphacep/vosk-api/releases/download/0.3.7/vosk-0.3.7-cp38-cp38-macosx_10_12_x86_64.whl

next release we'll update pip too.

**Comment by ptoche** [comment]
On `MacOS` and with a `miniconda3` setup I was able to install and run `vosk` with

    pip3 install -U https://github.com/alphacep/vosk-api/releases/download/0.3.7/vosk-0.3.7-cp38-cp38-macosx_10_12_x86_64.whl

or pip if pip3 is not symlinked. 

If you do it inside an environment, it will be installed inside that environment.

I do not have MacPorts (I'm on a clean install and haven't yet needed it).

**Comment by chenyuzhao98** [comment]
I ran into the exact same issue here. Thanks for the fix!

**Comment by AdiraBlumenthal** [comment]
On MacOS, how do I install the wheel with pip 20.1.1 instead of 20.0.2?

Error when running `pip3 install -U https://github.com/alphacep/vosk-api/releases/download/0.3.7/vosk-0.3.7-cp38-cp38-macosx_10_12_x86_64.whl`:

`pkg_resources.DistributionNotFound: The 'pip==20.0.2' distribution was not found and is required by the application`

**Comment by ptoche** [comment]
@AdiraBlumenthal, it's asking you to install/upgrade pip.

**Comment by nshmyrev** [comment]
This should be fixed now with 0.3.9
